guides/README.md

```
@@ -1,15 +1,70 @@
+| [GitHub Issues](https://github.com/AndyMik90/Auto-Claude/issues) | Bug reports |
+| [GitHub Discussions](https://github.com/AndyMik90/Auto-Claude/discussions) | Q&A |
+
+---
 
-## Quick Links
+## Documentation Overview
 
-- [Main README](../README.md) - Getting started (download and install)
-- [Contributing](../CONTRIBUTING.md) - How to contribute and run from source
-- [Changelog](../CHANGELOG.md) - Release history
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)